assemblyline_core/dispatching/dispatcher.py

```diff
@@ -722,75 +792,95 @@ def retry_error(self, task: SubmissionTask, sha256, service_name):
         task.running_services.pop((sha256, service_name), None)
         task.service_errors[(sha256, service_name)] = error_key
 
-        export_metrics_once(service_name, ServiceMetrics, dict(fail_nonrecoverable=1),
-                            counter_type='service', host='dispatcher')
+        export_metrics_once(service_name, ServiceMetrics, dict(fail_nonrecoverable=1), counter_type='service')
```

Comment:
Need some kind of host value for the metric to show up in dashboard, could use 'dispatcher'

Reply:
I fixed this in a couple places